packs/aws/sensors/sqs_sensor.py

```diff
@@ -32,9 +34,17 @@ def __init__(self, sensor_service, config=None, poll_interval=5):
                                            poll_interval=poll_interval)
 
     def setup(self):
-        self.input_queue = self._get_config_entry(key='keys ', prefix='sqs_sensor')
+        queues = self._get_config_entry(key='input_queues', prefix='sqs_sensor')
```

Comment:
Can you please handle 'input_queue' as well? Otherwise, this might break existing users. Add a LOG.warning() to indicate that the config is deprecated. We can remove it in next version.